### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by jihaneH**
### Leaderboard: final touches

- Integrated game: flappy bird
- Added design
- Added game score saved 
Capture d’écran 2022-01-27 à 20 29 12

### CODE REVIEW

**Comment by sheylaPozo** [comment]
## Status: ✔️ Project Approved ✔️ 
----

Hello @jihaneH  🔥 

_**highlights :**_ 
✔Excellent work in the commits
✔Proper git-flow
✔Excellent job at following all the instructions
✔ No linter errors
✔Proffessional documentation
✔Professional PR description
✔Really good job in the code and final details of it 

> # Excellent work on your project, it looks very professional and it's completed. Congrats on the good work!

surprised

Now you can celebrate! 🥇 🚀 👍 💯 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️
`There's nothing left to say but time to merge! 😊`

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html [hunk 1]
```diff
@@ -16,13 +16,21 @@ <h2>Recent scores</h2>
...
+          <small class="hidden"> Name & Score are required</small>
+          <input type="text" name="name" placeholder="Your name" required />
+          <input type="number" name="score" placeholder="Your score" id="lastScore" required />
+
           <button type="submit">Submit</button>
         </form>
       </section>
+      <canvas id="canvas" width="431" height="600"></canvas>
     </main>
   </body>
```
Comment (sheylaPozo) [inline dist/index.html:35]: - [ ] Your project is so beautifully set and done, great work on the code, and all you implemented, good job in adding flappy bird, it even works when you click on the flappy bird window and the fish moves and jumps which is something extra from the requirements, good job at the leader board showing score, the deployed project works excellent, good programming skills, I'm proud to say congratulations, your project's been approved ✔😁 without furthermore to say, you can now merge and enjoy your victory 🏆🥇✔💪😊**♻Keep doing a great job!! ✔**
ameri-iruma

### **✔✔Project Approved ✔✔**

**Review by sheylaPozo — APPROVED** [review]

File: dist/index.html (hunk 1 above)
Comment (jihaneH) [reply dist/index.html:35]: Hello, @sheylaPozo thank you for your post! 😊  
icegif-1433

Comment (sheylaPozo) [reply dist/index.html:35]: @jihaneH  you're welcome :D 💯

**Pull request merged**